### DISCUSSION TITLE
StockAlert Version

### DISCUSSION THREAD

**Discussion by xmpierce1** [Prince25/StockAlertBot#101] [General]
FYI.. I love this new version vs the previous one. I have been using it since the release and the intervals per store is a huge benefit on tracking GPU availability!  Using Cheerio is a major plus as well in terms of speed.

**Comment by Prince25** [comment]
Thank you! I agree. I started out just writing something that worked for me and then started stacking other things on top and it became messy and convoluted. I'm glad I took the time to redo everything and it's working much better. Way less issues than before. 😆
